sentry-uitest/sentry-uitest-android-benchmark/src/androidTest/java/io/sentry/uitest/android/benchmark/util/BenchmarkOperationResult.kt

```diff
@@ -0,0 +1,103 @@
+package io.sentry.uitest.android.benchmark.util
+
+import java.util.concurrent.TimeUnit
+
+/** Stores the results of a [BenchmarkOperation]. */
+internal data class BenchmarkOperationResult(
+    val avgCpuTimeMillis: Long,
+    val avgDroppedFrames: Double,
+    val avgDurationNanos: Long,
+    val avgFramesPerSecond: Int,
+    val operationName: String
+) {
+    /**
+     * Compare two [BenchmarkOperationResult], calculating increases of each parameter in percentage.
+     */
+    fun compare(other: BenchmarkOperationResult): BenchmarkResult {
+
+        // Measure average duration
+        val durationDiffNanos = avgDurationNanos - other.avgDurationNanos
+        val durationIncreasePercentage = durationDiffNanos * 100.0 / other.avgDurationNanos
+        println(
```

Comment:
Why do we have all of this `println` if this is going to run on CI/device farm? do we output this log somewhere?

Reply:
I removed few prints and made them comments. The idea is to start by printing the values, so that we can read them locally or from the logs on Saucelabs, and later on find a way to push these values somewhere else